<launch>
<group ns="pr2_ik_regression_test">    
      <param name="tip_name" value="r_wrist_roll_link" />     
      <param name="root_name" value="torso_lift_link" />  
</group>
</launch>